math/big/integer.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gxbig
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/big"
+)
+
+// Integer represents a integer value.
+type Integer struct {
+	big.Int
+
+	// used in hessian encoding/decoding
+	// You Should not use it in go
+	Signum int32
```

Comment:
the value of  these fields will always be zero， is that ok for java？

Reply:
these values will be and only be filled in IntegerSerializer.encObject